frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/tests/common.unit.spec.tsx

```diff
@@ -477,7 +475,7 @@ describe("Static Embed Setup phase", () => {
         ).toBeVisible();
 
         const link = within(
-          screen.getByLabelText("Playing with appearance options"),
+          screen.getByTestId("look-and-feel-settings"),
```

Comment:
There used to be a header in the red circle, but we remove it in the new design
![Screenshot 2024-06-17 at 9 41 07 PM](https://github.com/metabase/metabase/assets/1937582/d342720e-fa2b-4d76-9b3f-c76ecd766ad2)